core/storage/trie_pruner/impl/dummy_pruner.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_DUMMY_PRUNER_HPP
+#define KAGOME_DUMMY_PRUNER_HPP
```

Comment:
We agreed to replace include-guards by pragma-once while file changed. Lets do it for affected header-files.